code/modules/clothing/shoes/magboots.dm

```diff
@@ -102,3 +102,35 @@
 	icon_state = "syndiemag0"
 	base_state = "syndiemag"
 	species_fit = list(VOX_SHAPED)
+
+//Captain
+/obj/item/clothing/shoes/magboots/captain
```

Comment:
Remove the magboot stomp attack, since these boots don't have magnets. I think the variable is stomp_damage - just set it to 0 for this item

Reply:
They do have rather nasty spikes however, so I thought the magboot stomp would still be appropriate

Reply:
Found it - the variable for the magnet stomp is stomp_attack_power. The variable for kick damage is bonus_kick_damage - if you set stomp attack power to 0, you can still have stronger kicks by setting bonus_kick_damage